mllib/src/test/scala/org/apache/spark/ml/classification/GBTClassifierSuite.scala

```diff
@@ -66,10 +66,39 @@ class GBTClassifierSuite extends SparkFunSuite with MLlibTestSparkContext
     ParamsSuite.checkParams(new GBTClassifier)
     val model = new GBTClassificationModel("gbtc",
       Array(new DecisionTreeRegressionModel("dtr", new LeafNode(0.0, 0.0, null), 1)),
-      Array(1.0), 1)
+      Array(1.0), 1, 2)
     ParamsSuite.checkParams(model)
   }
 
+  test("Verify raw scores correspond to labels") {
```

Comment:
Could you take a look at [this test](https://github.com/apache/spark/blob/master/mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala#L314), and make it line up here? Specifically:

* compute probabilities manually from rawPrediction and ensure that it matches the probabilities column
* make sure that probabilities.argmax and rawPrediction.argmax equal the prediction
* make sure probabilities sum to one
* check the different code paths by unsetting some of the output columns